.circleci/config.yml

```diff
@@ -37,23 +35,31 @@ jobs:
             curl -L https://github.com/docker/compose/releases/download/1.12.0/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
             chmod +x /usr/local/bin/docker-compose
       - run:
-          name: Building image
-          command: docker build -t uwbpparamedics/uwblueprint-paramedics-web:0.0.1 -f Dockerfile.prod .
+          name: Add mock env file
+          command: touch paramedics-web.env
       - run: 
-          name: Run Docker compose
+          name: Start container via docker-compose
           command: |
             set -x
-            docker-compose -f docker-compose-prod.yaml up -d
-      - save_cache:
-          paths:
-            - node_modules
-          key: v1-dependencies-{{ checksum "package.json" }}
-  
+            docker-compose -f docker-compose-dev.yaml up -d
+  dev-lint:
+    docker: 
+      - image: uwbpparamedics/uwblueprint-paramedics-web:0.0.1
+    steps:
+      - checkout
+      - node/with-cache:
+          steps:
+            - run: npm install
+      - attach_workspace:
+          at: .
+      - run:
+          name: Lint files
+          command: npm run lint
+
 workflows:
   version: 2
-  build-test:
+  build-and-lint:
     jobs:
-      - install
-      - build:
-          requires:
-            - install
\ No newline at end of file
+      - prod-build
+      - dev-build
+      - dev-lint
```

Comment:
note that all three jobs are independent - there wasn't any reason to keep them linked together

Reply:
Hmm surprised that you can do `dev-lint` in parallel to `dev-build`

Reply:
How this works is still kinda murky to me but at this point I'm not focused on getting the best config possible - just one that does what we want relatively quickly (I'm sure there's further ways to optimize this but it's not worth the effort right now)